tests/plugins/PrefixLayer/basic/test.ktest

```diff
@@ -0,0 +1,235 @@
+RUN 1 cycle
+EXPECT no keyboard-report # no report after releasing PREFIX_B
+
+RUN 4 ms
+RELEASE CTRL
+RUN 1 cycle
+EXPECT keyboard-report empty # release Ctrl
+
+# ==============================================================================
+NAME Prefix layer same modifier second
+
+RUN 4 ms
+PRESS PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after pressing PREFIX_B
+
+RUN 4 ms
+PRESS CTRL
+RUN 1 cycle
+EXPECT keyboard-report Key_LCtrl # press Ctrl
+
+RUN 4 ms
+PRESS H
+RUN 1 cycle
+EXPECT keyboard-report Key_LCtrl Key_B # press B, Ctrl held
+EXPECT keyboard-report Key_LCtrl # release B, Ctrl held
+EXPECT keyboard-report empty # release Ctrl
+EXPECT keyboard-report Key_LCtrl # press Ctrl
+EXPECT keyboard-report Key_LCtrl Key_H # press H
+
+RUN 4 ms
+RELEASE H
+RUN 1 cycle
+EXPECT keyboard-report Key_LCtrl # release H
+
+RUN 4 ms
+RELEASE CTRL
+RUN 1 cycle
+EXPECT keyboard-report empty # release Ctrl
+
+RUN 4 ms
+RELEASE PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after releasing PREFIX_B
+
+# ==============================================================================
+NAME Prefix layer different modifier first
+
+RUN 4 ms
+PRESS SHIFT
+RUN 1 cycle
+EXPECT keyboard-report Key_LShift # press Shift
+
+RUN 4 ms
+PRESS PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after pressing PREFIX_B
+
+RUN 4 ms
+PRESS H
+RUN 1 cycle
+EXPECT keyboard-report Key_LCtrl # release pressed Shift and press Ctrl
+EXPECT keyboard-report Key_LCtrl Key_B # press B, Ctrl held
+EXPECT keyboard-report Key_LCtrl # release B, Ctrl held
+EXPECT keyboard-report empty # release Ctrl
+EXPECT keyboard-report Key_LShift # restoring pressed Shift
+EXPECT keyboard-report Key_LShift Key_H # press H
+
+RUN 4 ms
+RELEASE H
+RUN 1 cycle
+EXPECT keyboard-report Key_LShift # release H
+
+RUN 4 ms
+RELEASE PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after releasing PREFIX_B
+
+RUN 4 ms
+RELEASE SHIFT
+RUN 1 cycle
+EXPECT keyboard-report empty # release Ctrl
+
+# ==============================================================================
+NAME Prefix layer different modifier second
+
+RUN 4 ms
+PRESS PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after pressing PREFIX_B
+
+RUN 4 ms
+PRESS SHIFT
+RUN 1 cycle
+EXPECT keyboard-report Key_LShift # press Shift
+
+RUN 4 ms
+PRESS H
+RUN 1 cycle
+EXPECT keyboard-report Key_LCtrl # release pressed Shift and press Ctrl
+EXPECT keyboard-report Key_LCtrl Key_B # press B, Ctrl held
+EXPECT keyboard-report Key_LCtrl # release B, Ctrl held
+EXPECT keyboard-report empty # release Ctrl
+EXPECT keyboard-report Key_LShift # restoring pressed Shift
+EXPECT keyboard-report Key_LShift Key_H # press H
+
+RUN 4 ms
+RELEASE H
+RUN 1 cycle
+EXPECT keyboard-report Key_LShift # release H
+
+RUN 4 ms
+RELEASE SHIFT
+RUN 1 cycle
+EXPECT keyboard-report empty # release Ctrl
+
+RUN 4 ms
+RELEASE PREFIX_B
+RUN 1 cycle
+EXPECT no keyboard-report # no report after releasing PREFIX_B
```

Comment:
Sorry to ask for this, but it occurred to me that we ought to have a test for rollover from a printable character, _à la_:
1. press `K`
2. press `PREFIX_B`
3. press `H`
4. release `K`
5. release `PREFIX_B`
6. release `H`